util/perl/OpenSSL/paramnames.pm

```diff
@@ -220,8 +220,8 @@ my %params = (
 # DRBG call back parameters
     'DRBG_PARAM_ENTROPY_REQUIRED' =>        "entropy_required",
     'DRBG_PARAM_PREDICTION_RESISTANCE' =>   "prediction_resistance",
-    'DRBG_PARAM_MIN_LENGTH' =>              "minium_length",
-    'DRBG_PARAM_MAX_LENGTH' =>              "maxium_length",
+    'DRBG_PARAM_MIN_LENGTH' =>              "minimum_length",
+    'DRBG_PARAM_MAX_LENGTH' =>              "maximum_length",
```

Comment:
Hmm, is fixing these going to cause workflows to fail for people who are currently working round this with the incorrect spellings?

Reply:
Very likely.
Compatibility rules supreme.
Unfortunately.

Reply:
I think also the problems you are worried about could possibly happen.

Reply:
Is there some way to add the correct spellings in, with the current incorrect ones for backwards compatibility? Although I don't think that would count as trivial...